.devcontainer/Dockerfile

```diff
@@ -0,0 +1,21 @@
+FROM python:3.9
+
+ENV PYTHONDONTWRITEBYTECODE=1 \
+    PYTHONUNBUFFERED=1 \
+    USER=calitp
+
+# create $USER and home directory
+RUN useradd --create-home --shell /bin/bash $USER && \
+    chown -R $USER /home/$USER
+
+# switch to non-root $USER
+USER $USER
+
+# update PATH for local pip installs
+ENV PATH "$PATH:/home/$USER/.local/bin"
+
+RUN python -m pip install --upgrade pip && \
+    pip install pre-commit
```

Comment:
Not a big deal for now, but devcontainer requirements could be in a `requirements.txt` file and installed from there.

In the future if we need to worry about pinning dependencies or anything, having them in a `requirements.txt` allows for Dependabot and other tooling to more easily work with them.

Reply:
I think this is a good idea! It seems like we could do this in other projects as well (`benefits` and `eligibility-server`).